test/minitest5.bats

```diff
@@ -99,6 +110,20 @@ assert_test_queue_force_ordering() {
   assert_output_contains "MiniTestFailure#test_fail"
 }
 
+@test "multi-master remote master fails when a test fails" {
+  export FAIL=1
+  export TEST_QUEUE_RELAY_TOKEN=$(date | cksum | cut -d' ' -f1)
+  TEST_QUEUE_SOCKET=0.0.0.0:12345 bundle exec minitest-queue ./test/samples/sample_minitest5.rb || true &
+  sleep 0.1
+  TEST_QUEUE_RELAY=0.0.0.0:12345 run bundle exec minitest-queue ./test/samples/sample_minitest5.rb
+  wait
+
+  assert_status 1
```

Comment:
I think this will sometimes fail due to the issue I mentioned. There's no guarantee that the remote master will run the failing test. If it doesn't, its exit code will be 0 (and it won't contain any failure output). We need a way to ensure that the right master fails. Something like this might work inside a test:

```ruby
def test_foo
  whoami =
    if ENV["TEST_QUEUE_SOCKET"]
      "central"
    elsif ENV["TEST_QUEUE_RELAY"]
      "remote"
    end
  refute ENV["MULTI_MASTER_FAIL"] == whoami
end
```

I.e., if you set `MULTI_MASTER_FAIL=central`, the central master will fail. If you set `MULTI_MASTER_FAIL=remote`, the remote master will fail. If you put this new test inside the `MiniTestSleep` suites then it will be highly likely to run in both masters and fail or pass appropriately.